tests/FakeItEasy.Specs/ConfigurationSpecs.cs

```diff
@@ -900,5 +901,16 @@ protected override IFoo Create()
                 return Instance;
             }
         }
+
+        public class VoidOptionsBuilder : IFakeOptionsBuilder
+        {
+            public Priority Priority => Priority.Default;
+
+            public bool CanBuildOptionsForFakeOfType(Type type) => type == typeof(void);
+
+            public void BuildOptions(Type typeOfFake, IFakeOptions options)
+            {
+            }
+        }

```

Comment:
I didn't actually need to change any scenario. We already had some that failed by just adding this options builder.

Reply:
Interesting. I'm torn between asking for an explicit test (what if all the other ones get removed? But I think I'm being a little overcautious) and a code-level comment to indicate why this builder exists, since it won't be obvious who's using it.

Reply:
I'll see if I can add an explicit scenario, but I suspect it will do mostly the same thing as one of the existing ones... If that's the case, I'll just add a comment.